enhancements/windows-containers/health-management.md

```diff
@@ -66,18 +66,24 @@ configuration of a Windows instance, to ensure that it can become a worker Node.
 
 To accomplish the goals listed in this enhancement, I am proposing that WMCB be converted into a daemon,
 Windows Instance Config Daemon (WICD). WICD will have the following responsibilities:
-* WICD maintains current responsibilities of bootstrapping kubelet through the `bootstrap` command.
+* As part of the `bootstrap` phase, WICD will configure the [containerd](container-runtime-containerd.md) runtime and 
+  start it as a Windows service, if its configuration is detailed in the services ConfigMap.
+* WICD will also maintain current responsibilities of starting kubelet as a service through the `bootstrap` command. 
+  WICD will fully configure kubelet in one shot, as everything required will be available in the services ConfigMap.
 * Using the configuration provided by a ConfigMap created by WMCO, WICD maintains the state of Windows Services on the 
   instance.
 * WICD reverts all changes made to an instance when run with the `cleanup` command, and also deletes the Node
   object.
 
 To enable this, WMCO will have these changes in responsibility:
 * WMCO will no longer configure the Windows services on Windows instances, except for WICD.
+* WMCO will take over the parsing of the worker ignition file. Note that WMCO will now be able to provide all the 
+  information needed to fully configure the kubelet service upon its creation as 
+  [CNI configuration is no longer provided to kubelet](https://github.com/kubernetes/kubernetes/pull/106907#).
```

Comment:
Lets make sure we mention that WMCO will copy over the files contained within the ignition file, such as the bootstrap kubeconfig and the kubelet-ca.crt https://github.com/openshift/windows-machine-config-bootstrapper/blob/b9612c9cdf99acb1184810817b56b4b1a9ef8b95/pkg/bootstrapper/bootstrapper.go#L467-L472

Reply:
Sure, added directly below this line.